src/doc/book/closures.md

```diff
@@ -492,12 +492,12 @@ fn factory() -> Box<Fn(i32) -> i32> {
 
     Box::new(move |x| x + num)
 }
-# fn main() {
+fn main() {
 let f = factory();
 
 let answer = f(1);
 assert_eq!(6, answer);
-# }
+}
```

Comment:
To be honest I am a bit confused by the rollups below. Not yet very good at git. 😄